script/Ciel/Day 14/14_01_CIEL14_1.txt

```diff
@@ -486,17 +527,27 @@ How many seconds? How many minutes... will this... pain... go on... for?
 [sha:c082dfd9b9823b6d3670c48016083d0ed4ce8f53]{
 -- Page 29, Offset 37100.
 -- いたい。どうにかしようと、ナイフで掌をザクザクと刺してみた。ダメだ。ぜんぜんイタク、ない。そうか。たぶんきっと線の　セイだ。眼鏡。眼鏡をかければ、いつものように、こんな異常は、収まって
-It hurts. In one way or another, I try to stab away at my palm with my knife. It's no good. It doesn't hurt at all. I see. It must be because of the lines. My glasses. If I put on my glasses, this kind of abnormality will go away, as always
+It hurts.
+ In a desperate attempt to do %{i}something%{/i}, I stab the palm of my hand over and over again with my knife.
+ It's no use.
+ It doesn't hurt at all.
+ Wait... of course.
+ This is all because of the lines.
+ My glasses――this will all go away if I put on my glasses, just like it always does.
```

Comment:
Now this one needs a stylistic makeover
>It hurts.
>Just to do something,%{i}anything%{/i}, I stab my palm over and over again with my knife.
>It doesn't　work.
>It doesn't　h  u  rt  a t  all.
>Of　course.
>This is all be c  a us   e 　of th  e 　lines.
>My glasses——this will all go away　if I put 　on   m y   glasses, just  l  i ke it　　always does.